changelog.d/20220922_151311_kevin_sqlite_adapter_connection_params.rst

```diff
@@ -0,0 +1 @@
+* Add `connect_params` passthrough dictionary to SQLiteAdaptor, enabling fine-tuning of DB connection parameters (:pr:`NUMBER`)
```

Comment:
```suggestion
* Add ``connect_params`` passthrough dictionary to ``SQLiteAdapter``, enabling fine-tuning of DB connection parameters (:pr:`NUMBER`)
```

Since this is an RST fragment, we need double-backticks here. And I would code-quote the classname as well.